### ISSUE TITLE
style_directives checks in comments too?

### ISSUE THREAD

**Issue by Timmmm** [dalance/svlint#305]
If you have this code

```
// `ifdef FOO
```

Then the `style_directives` lint will complain `Remove whitespace preceeding compiler directive.` about the `ifdef` in the comment. That doesn't seem right.

It seems like the rule is purely regex based so it can't be sophisticated enough to detect this. The current regex is

```
format!("^(.+)`({})", keywords)
```

This is going to be very prone to false-positives (e.g. markdown in comments for example). I would recommend making it tighter, i.e..

```
format!("^(\s+)`({})", keywords)
```

**Comment by Timmmm** [comment]
Yeah that seems reasonable.
